Apigee/Mint/BankDetail.php

```diff
@@ -152,16 +152,22 @@ public function save($save_method)
         } else {
             $baseUrl = '/mint/organizations/' . rawurlencode($this->config->orgName) . '/bank-details/' . $this->id;
             $this->setBaseUrl($baseUrl);
-            $this->put(null, $this->__toString());
-            $this->restoreBaseUrl();
+            try {
+              $this->put(null, $this->__toString());
+            } finally {
```

Comment:
`finally` should be on a new line, see https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct. Lots of places below as well.

Reply:
This repo should be using [PSR-2](https://www.php-fig.org/psr/psr-2/) actually, since it is separate from Drupal.

Reply:
Doh, I get a little fixated on Drupal sometimes :)